H5P.BranchedVideo/branched-video.js

```diff
@@ -0,0 +1,847 @@
+var H5P = H5P || {};
+
+H5P.BranchedVideo = (function ($) {
+  /**
+   * Constructor function.
+   */
+  function C(options, id) {
+    this.$ = $(this);
+    // Extend defaults with provided options
+    this.options = $.extend(true, {}, {
+        branchedVideos: [
+          {
+            slug: '',
+            title: '',
+            description: '',
+            length: 0,
+            sourceFiles: {},
+            subBranches: {}
+          }
+        ],
```

Comment:
I don't think we would be able to determine which is the main branch without the mainBranchSlug H5P text object.

Reply:
I meant instead of `branchedVideos: [
          {
            slug: '',
            title: '',
            description: '',
            length: 0,
            sourceFiles: {},
            subBranches: {}
          }
        ]`